lighthouse-core/gather/driver.js

```diff
@@ -946,8 +911,8 @@ class Driver {
   endTrace() {
     return new Promise((resolve, reject) => {
       // When the tracing has ended this will fire with a stream handle.
-      this.once('Tracing.tracingComplete', streamHandle => {
-        this._readTraceFromStream(streamHandle)
+      this.once('Tracing.tracingComplete', completeEvent => {
```

Comment:
note that these are now type checked (not `any`!) but they are also inferred correctly from the event string, so no annotation needed :)